selenium_tests/pin/page_objects/pin_page.py

```diff
@@ -0,0 +1,67 @@
+"""
+This file models some of the page objects on the Manage People Search Page.  (find_user.html)
+"""
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.common.by import By
+from selenium_tests.pin.page_objects.base_page_object import PinBasePageObject
+import abc
+
+class PinPageLocators(object):
+    # List of WebElements found on Pin Page (Locators)
+    LOGIN_TYPE_XID_RADIO_BUTTON = (By.ID, "compositeAuthenticationSourceType3")
```

Comment:
We should not be looking for the third radio button, but for the XID radio button.  Look for the XID text instead of the third radio button, which might change.